lessons/08-tests-tasks/README.md

```diff
@@ -155,14 +173,17 @@ The `cover-mocha` script also shows how to generate server-side code coverage us
 
 # Lesson 8 In-Class Exercises
 
-## Testing:
+### Cat App Testing:
 We've built a testing framework for the cat app with a few example tests (the app and test suite are in the subdirectory `inclass`). Follow the instructions in `inclass/README.md` to get set up, then...
 - Add tests for the rest of the cat app.
   - We've set up a tool called supertest for testing the routes modules; those tests live in `test/server/appSpec.js`. Check out the supertest documentation on Github or npm to learn more.
 - "Break" the cat app source code in several places to make sure your tests aren't passing when there are problems.
 - Trade cat apps with someone else and try to create a bug in the existing cat app code which their tests won't catch.
 
-## Task running:
+### Burger App Testing:
```

Comment:
Can we put this one before the Cat App Testing section, since we're doing it as a group whiteboard activity?